### PR TITLE
Fix transient hostname, enable Kconfig switches for Firecracker shutdown

### PR DESCRIPTION
**Pull request by twelho**

### CODE REVIEW

File: firecracker-vms.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (luxas) [inline firecracker-vms.iml]: please remove this file

File: ubuntu-base/Dockerfile [hunk 2]
```diff
@@ -26,13 +26,15 @@ RUN mkdir -p /etc/systemd/system/sockets.target.wants /etc/systemd/system/networ
...
 # Install some utilities, which are needed in most installations
+# PolicyKit is required for systemd-networkd to set the transient hostname
 RUN apt-get update && \
     apt-get -y install --no-install-recommends \
+      policykit-1 \
       bash-completion \
       command-not-found \
       openssh-server \
       rng-tools \
-      nano \
```
Comment (luxas) [inline ubuntu-base/Dockerfile:35]: I like nano :)
Please keep both

**Review by luxas — COMMENTED** [review]
Thanks, small nits only

File: firecracker-vms.iml (hunk 1 above)
Comment (twelho) [reply firecracker-vms.iml]: Deleted in 112c9894452b36afc9716ffd7aa2514800d357a8.

File: ubuntu-base/Dockerfile (hunk 2 above)
Comment (twelho) [reply ubuntu-base/Dockerfile:35]: nano gets stuck in Firecracker, you cannot save or quit and need to kill the VM.

**Review by luxas — COMMENTED** [review]
LGTM

**Pull request merged**